### DISCUSSION TITLE
HTTP Dasbhard is Blank

### DISCUSSION THREAD

**Discussion by jrm360seclab** [Security-Onion-Solutions/securityonion#8658] [Unsupported Versions]
Good morning all, 

How do I get data for the HTTP dashboard. I have gone as far back as weeks but I have nothing. How do I capture the HTTP/HTTPS traffic so I can see the same type of data you have on this sample dashboard.
https://user-images.githubusercontent.com/1659467/174881466-7c6cd85b-8bcd-403a-a7cd-2ef591f6902d.png

**Comment by dougburks** [comment]
How did you install and configure Security Onion?

Have you verified that you are receiving HTTP traffic from your tap or span port?

Are you using Zeek or Suricata for metadata? If Zeek, have you checked `/nsm/zeek/logs/current/` to see if it's writing `http.log` correctly?

Is Suricata generating any NIDS alerts for HTTP traffic?

This troubleshooting checklist is for Suricata alerts, but some of the steps may be helpful:
https://docs.securityonion.net/en/2.3/suricata.html#troubleshooting-alerts

**Reply by jrm360seclab** [reply]
Thank  you @dougburks 
  mdengine: 'ZEEK'
  ids: 'Suricata'


I checked the /nsm/zeek/logs/current/ and I do not have http.log file in there. 
These are the logs in there:

broker.log  capture_loss.log  conn.log  notice.log  stats.log  stderr.log  stdout.log

**Reply by dougburks** [reply]
Are you collecting traffic from a tap or span port?

Have you verified that you are receiving HTTP traffic from that tap or span port using `tcpdump` or a similar tool?

> I checked the /nsm/zeek/logs/current/ and I do not have http.log file in there.
These are the logs in there:
broker.log capture_loss.log conn.log notice.log stats.log stderr.log stdout.log

Have you checked stdout.log, stderr.log, notice.log for any additional clues?

What kind of connections are shown in conn.log?
